tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md

```diff
@@ -0,0 +1,344 @@
+![image](https://github.com/ideafest/tutorials/blob/master/image27.png)
+
+8. We can skip the section “Additional settings” by clicking the “NEXT: CLUSTER NAME” button since we will keep the default settings.
+
+![image](https://github.com/ideafest/tutorials/blob/master/image35.png)
+
+9. Under “Cluster Name”- Provide a Cluster Name. For this demo, we will keep the default of “Cluster0”. Click on “Create Cluster” button at the bottom of the page.
+
+![image](https://github.com/ideafest/tutorials/blob/master/image38.png)
+
+10. Our cluster is spinning up...
+
+![image](https://github.com/ideafest/tutorials/blob/master/image36.png)
+
+11. While this spins up, lets click on “Security” tab and then click on “Add New User”.  
+
+![image](https://github.com/ideafest/tutorials/blob/master/image22.png)
+
+12. Leave the default user as of “admin” and select a secure password.  Record your user name and password in a safe location for reference later. Under “User Privileges”, select     “Atlas admin” and click on the “Add User” button to complete this section.
+
+![image](https://github.com/ideafest/tutorials/blob/master/image16.png)
+
+13. Once done, we will see screen similar to this…
+
+![image](https://github.com/ideafest/tutorials/blob/master/image32.png)
+
+14. Under Security tab, select “IP Whitelist” and click on “Add IP Address”.
+
+![image](https://github.com/ideafest/tutorials/blob/master/image26.png)
+
+15. Select “Allow Access from Anywhere” for the purpose of this demo and click on “Confirm”.  Note:  When actually putting something into production, you will want to narrow the scope of where your database can be accessed and specify a specific IP address/CIDR block.
+
+![image](https://github.com/ideafest/tutorials/blob/master/image20.png)
+
+16. Go to “Overview” tab and click on “Connect” button.
+
+![image](https://github.com/ideafest/tutorials/blob/master/image42.png)
+
+17. A window will open. Select “Connect Your Application”
+
+![image](https://github.com/ideafest/tutorials/blob/master/image19.png)
+
+18. Click on “I am using driver 3.4 or earlier” and copy the connection string and keep it in a text file. We will be using it in our node JS application to connect to MongoDB Atlas in Part 2 of this document. Close the pop-up dialogue
+  
+![image](https://github.com/ideafest/tutorials/blob/master/image40.png)
+![image](https://github.com/ideafest/tutorials/blob/master/image41.png)
+
+### Part 2: Configuring our Google Cloud Platform Account
+
+1. Go to cloud.google.com and login with your google account. If you don't have a google account, please create a free trial account by following instructions at this [link](https://console.cloud.google.com/freetrial).
```

Comment:
Capitalize "Google" throughout the page.

Delete "please" throughout the page.